cmd/agent/app/integrations.go

```diff
@@ -115,6 +120,46 @@ var showCmd = &cobra.Command{
 	RunE:  show,
 }
 
+func detectEnvironment() {
+	rootDir, _ = executable.Folder()
```

Comment:
This will indeed resolve all symlinks - so it will take us to the agent executable from `/usr/bin/datadog-agent` 👍

Reply:
Also, the path traversal below should work well across all platforms.